mirror_pso_containers.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Script to mirror PSO required containers into a local Registry
+
+# Ensure you have logged in to your local registry using 'docker login' before using this script
+
+REGISTRY_URL=<your local registry address, including port>
+
+LIST=(
+quay.io/k8scsi/csi-provisioner:v1.6.0
```

Comment:
This means we need to check this list every time we cut a new release. Team members need to be aware of it.

Reply:
I did reference this in the PR description